src/sentry/api/endpoints/organization_events_spans_performance.py

```diff
@@ -598,6 +598,11 @@ def query_example_transactions(
         [
             Condition(Function("has", [builder.column("spans_op"), span.op]), Op.EQ, 1),
             Condition(Function("has", [builder.column("spans_group"), span.group]), Op.EQ, 1),
+            Condition(
+                builder.resolve_span_function("count", span, "example_count"),
+                Op.GT,
+                0,
+            ),
```

Comment:
Can you add a test case for this in https://github.com/getsentry/sentry/blob/master/tests/snuba/api/endpoints/test_organization_events_spans_performance.py ?

Reply:
added! let me know what you think :)